source/extensions/filters/network/common/redis/supported_commands.h

```diff
@@ -109,8 +109,8 @@ struct SupportedCommands {
    * @return commands which handle Redis commands without keys.
    */
   static const absl::flat_hash_set<std::string>& allShardCommands() {
-    CONSTRUCT_ON_FIRST_USE(absl::flat_hash_set<std::string>, "script", "flushall", "pubsub", "keys", "slowlog", "config","client","info","select");
-    }
+    CONSTRUCT_ON_FIRST_USE(absl::flat_hash_set<std::string>, "script", "flushall", "pubsub", "keys", "slowlog", "config","client","info","select", "subscribe", "psubscribe", "punsubscribe", "unsubscribe");
```

Comment:
why should subscribe be an all shard commands

Reply:
why not create another hash map called unsubscribe commands , so that it can also be treated as all shard command, any some complex il else can also be avoided